modules/administration/pages/content-staging.adoc

```diff
@@ -0,0 +1,67 @@
+[[content-staging]]
+= Content Staging
+
+
+Staging is used by clients to download packages in advance, before they are installed.
+This allows package installation to begin as soon as it is scheduled, which can reduce the amount of time required for a maintenance window.
+
+
+== Enable Content Staging
+
+You can manage content staging across your entire organization.
+In the {productname} {webui}, navigate to menu:Admin[Organizations] to see a list of available organizations.
+Click the name of an organization, and check the [guimenu]``Enable Staging Contents`` box to allow clients in this organization to stage package data.
+
+[NOTE]
+====
+You must be logged in as a {productname} administrator to create and manage organizations.
+====
+
+You can also enable staging at the command prompt by editing [path]``/etc/sysconfig/rhn/up2date``, and adding or editing these lines:
+
+----
+stagingContent=1
+stagingContentWindow=24
+----
+
+////
+2018-12-10, ke: /etc/sysconfig/rhn/up2date still exists. @renner confirmed some tools use it (at least, trad. client).  To be renamed in the future.
+////
+
+The ``stagingContentWindow`` parameter is a time value expressed in hours and determines when downloading will start.
+It is the number of hours before the scheduled installation or update time.
+In this example, content will be downloaded 24 hours before the installation time.
+The start time for download depends on the selected contact method for a system.
+The assigned contact method sets the time for when the next [command]``mgr_check`` will be executed.
+
+Next time an action is scheduled, packages are automatically downloaded, but not installed.
+At the scheduled time, the staged packages are installed.
+
+
+
+== Configure Content Staging
+
+There are two parameters used to configure content staging:
+
+* [parameter]``salt_content_staging_advance`` is the advance time for the content staging window to open, in hours.
+This is the number of hours before installation starts, that package downloads can begin.
+* [parameter]``salt_content_staging_window`` is the duration of the content staging window, in hours.
+This is the amount of time clients have to stage packages before installation begins.
+
+For example, if [parameter]``salt_content_staging_advance`` is set to six hours, and [parameter]``salt_content_staging_window`` is set to two hours, the staging window will open six hours before the installation time, and remain open for two hours.
+No packages will be downloaded in the four remaining hours until installation starts.
+
+If you set the same value for both [parameter]``salt_content_staging_advance`` and [parameter]``salt_content_staging_window`` packages will be able to be downloaded until installation begins.
+
+Configure the content staging parameters in [path]``/usr/share/rhn/config-defaults/rhn_java.conf``.
+
+Default values:
+
+* [path]``salt_content_staging_advance: 8 hours``
```

Comment:
```suggestion
* [literal]``salt_content_staging_advance: 8 hours``
```

Reply:
i thought i changed these to [parameter] actually :confused: